File: scripts/script_tust.jl
include("convert_base.jl")

cycles = ["2023-2024","2024-2025","2025-2026","2026-2027","2027-2028","2028-2029","2029-2030","2030-2031", "2031-2032"]
tust_path = raw"D:\dev\auto_tust\cases\BasePSR"

#convert_ger(tust_path,cycles)
convert_dc(tust_path, cycles)
#convert_tuh(tust_path,cycles)
#convert_nos(tust_path,cycles)
#convert_all_base(tust_path, cycles)

File: scripts/script_tust.py
import autotust_v60


case_path = r"C:\Users\matheusduarte\Desktop\BasePSR_Alt1e2A_RAP_Leilao_NovaTransicao"
cycles = ["2022-2023","2023-2024","2024-2025","2025-2026","2026-2027","2027-2028","2028-2029","2029-2030","2030-2031"]


#convert_ger(tust_path,cycles)
#convert_dc(tust_path,cycles)
#convert_tuh(tust_path,cycles)
#convert_nos(tust_path,cycles)
autotust_v60.run(case_path, cycles)